region/brinstar/pink/Pink Brinstar Hopper Room.json

```diff
@@ -249,17 +286,128 @@
+        {"enemyDamage": {
+          "enemy": "Sidehopper",
+          "type": "contact",
+          "hits": 1
+        }},
+        "canNeutralDamageBoost",
+        "canPrepareForNextRoom",
+        "HiJump",
+        "canOffScreenSuperShot"
+      ],
+      "clearsObstacles": ["B"],
+      "reusableRoomwideNotable": "Pink Hopper Reverse Gate Glitch",
+      "note": [
+        "Use damage from the small sidehopper to align against the left wall.",
+        "Jump straight up and shoot with the correct timing.",
         "This strat involves shooting a Super diagonally from the correct height while flush against the left wall.",
         "Its acceleration will cause it to clip into the blue gate off-screen."
       ],
+      "devNote": [
+        "The reverse gate glitch pretty much requires killing the enemies first, so it's required here.",
+        "Springball jumps didn't look possible but was close.",
+        "IBJ didn't look possible."
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "Pink Hopper Reverse Gate Glitch (SpaceJump)",
+      "notable": true,
+      "requires": [
+        {"obstaclesCleared": ["A"]},
+        {"or": [
+          "canTrickyJump",
+          "h_canBackIntoCorner"
+        ]},
+        "canStationarySpinJump",
+        "SpaceJump",
+        "canOffScreenSuperShot"
+      ],
+      "clearsObstacles": ["B"],
+      "reusableRoomwideNotable": "Pink Hopper Reverse Gate Glitch",
+      "note": [
+        "This strat involves shooting a Super diagonally from the correct height while flush against the left wall.",
+        "Its acceleration will cause it to clip into the blue gate off-screen.",
+        "Use a stationary spinjump, or spinjump against the wall and turn around without moving; and shoot with the same correct timing."
+      ],
+      "devNote": [
+        "The reverse gate glitch pretty much requires killing the enemies first, so it's required here."
+      ]
+    },
+    {
+      "link": [1, 1],
+      "name": "Pink Hopper Reverse Gate Glitch (Speedy Jump)",
+      "notable": true,
+      "requires": [
+        {"obstaclesCleared": ["A"]},
+        "canTrickyDashJump",
+        "canOffScreenSuperShot"
+      ],
+      "clearsObstacles": ["B"],
+      "reusableRoomwideNotable": "Pink Hopper Reverse Gate Glitch",
+      "note": [
+        "This strat involves shooting a Super diagonally from the correct height while flush against the left wall.",
+        "Its acceleration will cause it to clip into the blue gate off-screen.",
+        "Build run speed and jump into the left wall, turning around without changing Samus' pixel position."
+      ],
+      "devNote": "The reverse gate glitch requires killing the enemies first, so it's required here."
+    },
+    {
+      "link": [1, 1],
+      "name": "Pink Hopper Reverse Gate Glitch (Door Frame Jump)",
+      "notable": true,
+      "requires": [
+        {"obstaclesCleared": ["A"]},
+        {"doorUnlockedAtNode": 1},
+        "canTrickyJump",
```

Comment:
I notice that some of the earlier strats dont always require a tricky jump (HiJump/Space Jump having an `h_canBackIntoCorner` alternative). Should those also have a door unlocked variant?